### DISCUSSION TITLE
Replace / enhance "x <time unit> ago" by actual timestamp

### DISCUSSION THREAD

**Discussion by lukaseder** [community/community#7184] [Projects and Issues]
I can see how displaying the time diff between now and when something happened can be useful, occasionally, but I very much rather like to see the actual timestamp pretty much everywhere in the github UIs. For example, when looking at a milestone:

![image](https://user-images.githubusercontent.com/734593/140027450-9794331a-4f03-4958-92df-1a0f9c782d95.png)

I'd like to see which ones of these issues were fixed after a given date (e.g. a specific nightly build). I have to do some mental date time arithmetic, which is so painful as we all know (plus, we switched from CEST to CET recently).

Why not display the actual timestamp instead, or in addition to the diff? In fact, this isn't done consistently. The "opened" info is shown as I expected:

![image](https://user-images.githubusercontent.com/734593/140043428-8e88bf74-fdcd-4b75-94fd-bdd452d3cffd.png)

**Comment by ppalaga** [comment]
Yep, definitely makes sense, especially when comparing commit dates with some specific date, such as tag date. If the tag date is 2021-01-01 10:00:00 then it is much easier to decide about  2021-01-01 09:59:59 than about "x months ago".

**Comment by olafurpg** [comment]
If you hover over "X days ago" you get a tooltip that displays the full date and timestamp. This has always been sufficient for my own needs when I've wanted to know the exact timestamp. I like that the relative human readable time is visible by default.

<img width="565" alt="image" src="https://user-images.githubusercontent.com/1408093/140030796-a0d52ac2-a98e-4707-a4fa-44b0d6ff8fe3.png">

**Reply by lukaseder** [reply]
Good to know as a workaround.

**Reply by uschindler** [reply]
Pro tip: JIRA issue tracker which has exact same problem. Hovering helps.

**Reply by GfEW** [reply]
To each their own. For me, hovering doesn't solve the problem, even if it's a satisfying workaround for some.

If you need to see and compare exact times, those "friendly" timestamps are hilariously vague and of little use.

Moreover, on many mobile setups, hovering isn't available. Even on desktop (or mobile with mouse), hovering every single "friendly" timestamp just to briefly display what should be visible everywhere right away, is needlessly time consuming and tedious.

And even _if_ you're willing to jump through those hoops on a daily basis, seeing at most one exact timestamp at a time makes understanding their temporal relations much harder than it should be. It doesn't make sense to prevent the user from comparing exact times vis-a-vis.

**Comment by vegegoku** [comment]
This seems to me like a user preference can be changed in the profile settings..

**Comment by Lilred78** [comment]
I don't know what to do with myself I don't know what this is

**Comment by Lilred78** [comment]
I don't know what to do with myself I don't know what to do with
